crates/exex/exex/src/dyn_context.rs

```diff
@@ -0,0 +1,56 @@
+//! Mirrored version of [`ExExContext`](`crate::ExExContext`)
+//! without generic abstraction over [Node](`reth_node_api::FullNodeComponents`)
+
+use std::sync::Arc;
+
+use reth_chainspec::{EthChainSpec, Head};
+use reth_node_api::FullNodeComponents;
+use reth_node_core::node_config::NodeConfig;
+use tokio::sync::mpsc;
+
+use crate::{ExExContext, ExExEvent};
+
+// TODO(0xurb) - add `node` and `notifications` after abstractions
```

Comment:
we already have `notifications` as a trait now, can we add integrate it here?

Reply:
sure